lib/fastlane/actions/carthage.rb

```diff
@@ -26,39 +29,50 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :update,
-                                       env_name: "FL_CARTHAGE_UPDATE",
-                                       description: "Update the the project's dependencies",
-                                       is_string: false,
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
                                        optional: true,
-                                       default_value: false,
```

Comment:
Instead of having an if/else on the implementation, you could make "bootstrap" the default value here :)